settings.gradle.kts

```diff
@@ -57,33 +56,30 @@ gradleEnterprise {
 		}
 
 		obfuscation {
-			if (isCiServer) {
+			if (buildParameters.ci) {
 				username { "github" }
 			} else {
 				hostname { null }
 				ipAddresses { emptyList() }
 			}
 		}
 
-		val enableTestDistribution = providers.gradleProperty("enableTestDistribution")
-			.map(String::toBoolean)
-			.getOrElse(false)
-		if (enableTestDistribution) {
+		if (buildParameters.enterprise.enableTestDistribution) {
```

Comment:
❌ I'm afraid this isn't the only place this parameter is used. It's also used in `buildSrc`:
https://github.com/junit-team/junit5/blob/fe1997419d4fd5b0424a5e992a26f1826d2250c2/buildSrc/src/main/kotlin/testing-conventions.gradle.kts#L24

Reply:
I'm aware of this but wanted to have feedback on the direction first before fixing this. See below :o)